plugin/src/main/kotlin/org/gradleweaver/plugins/jlink/CompressionLevel.kt

```diff
@@ -0,0 +1,21 @@
+package org.gradleweaver.plugins.jlink
+
+/**
+ * Options for the levels of compression available for generated runtime images.
+ */
+enum class CompressionLevel {
```

Comment:
Can you link to documentation describing these each in more detail?

Reply:
https://docs.oracle.com/javase/9/tools/jlink.htm

Reply:
This is also linked in the README

Reply:
Can you link to the exact section?
https://docs.oracle.com/javase/9/tools/jlink.htm#GUID-CECAC52B-CFEE-46CB-8166-F17A8E9280E9__GUID-DF03D0FC-9390-4C17-BEEA-ECC2EA9B94F2

Reply:
The page doesn't support linking to the individual sections